### PR TITLE
Update README so pypi download and python code examples are at the top

### PR DESCRIPTION
**Pull request by bargazit**

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -26,10 +24,10 @@ def load_args():
...
     parser.add_argument("--max_len", type=int, default=16,
                         help="max length of tokens in bytes. Default is 16")
-    parser.add_argument("--workers", type=int, default=max(1, mp.cpu_count() - 1),
-                        help="number of worker threads to use. Default is max(1, mp.cpu_count()-1)")
-    parser.add_argument("--random_seed", type=int, default=random.randint(1, 10000),
-                        help="random seed value. Default is random.randint(1, 10000)")
+    parser.add_argument("--workers", type=int, default=1,
+                        help="number of worker threads to use. Default is 1")
+    parser.add_argument("--random_seed", type=int, default=1234,
+                        help="random seed value. Default is 1234")
```
Comment (yuvalpinter) [inline src/main.py]: Same here, why this default?

File: README.md
Comment (yuvalpinter) [inline README.md:1]: Please add an explicit `pip install` block. It's still not evident.

File: README.md [hunk 2]
```diff
@@ -41,8 +53,8 @@ Default override arguments:
 - `partial_corpus_filepath`: where to create / load partial corpus file. Default is `''` for creating partial corpus under 'data' folder.
 - `partial_corpus_line_number`: number of lines for partial corpus - in thousands. Default is `1000`.
 - `max_len`: max length of tokens in bytes. Default is `16`.
-- `workers`: number of worker threads to use. Default is `max(1, mp.cpu_count()-1)`.
-- `random_seed`: random seed value. Default is `random.randint(1, 1000)`.
+- `workers`: number of worker threads to use. Default is `1`.
+- `random_seed`: random seed value. Default is `1234`.
```
Comment (yuvalpinter) [inline README.md]: Why is the default now a specific number?

File: README.md
Comment (bargazit) [reply README.md:1]: Found in lines 17-19.

File: README.md (hunk 2 above)
Comment (bargazit) [reply README.md]: Needed to align between execution via shell/main vs via direct usage from Python. As discussed will change random seed to 692653.

**Pull request merged**